specs/eip4844/polynomial-commitments.md

```diff
@@ -222,44 +223,34 @@ def blob_to_polynomial(blob: Blob) -> Polynomial:
     return polynomial
 ```
 
-#### `compute_challenges`
+#### `compute_challenge`
 
 ```python
-def compute_challenges(polynomials: Sequence[Polynomial],
-                       commitments: Sequence[KZGCommitment]) -> Tuple[Sequence[BLSFieldElement], BLSFieldElement]:
+def compute_challenge(polynomial: Polynomial,
```

Comment:
Wouldn't it be simpler if we switched `compute_challenge()` to take a Blob as an argument (instead of a Polynomial) since it's already a ByteVector?

We could just do `data += blob` in that case.